skbio/sequence/_grammared_sequence.py

```diff
@@ -809,12 +808,21 @@ def to_definites(self, degenerate="wild", noncanonical=True):
                     f"Invalid character '{degenerate}' in sequence. Character must "
                     f"be within sequence alphabet: {self.alphabet}"
                 )
+        elif isinstance(degenerate, str) and len(degenerate) == 0:
```

Comment:
This can be `elif degenerate == "":`, although I think there is a better method (see below).